dataland-frontend/src/components/forms/parts/UploadReports.vue

```diff
@@ -1,20 +1,26 @@
 <template>
-  <div class="col-3 p-3 topicLabel">
+  <div v-if="isEuTaxonomy" class="col-3 p-3 topicLabel">
     <h4 id="uploadReports" class="anchor title">Upload company reports</h4>
     <p>Please upload all relevant reports for this dataset in the PDF format.</p>
   </div>
   <!-- Select company reports -->
-  <div class="col-9 formFields">
+  <div v-if="isEuTaxonomy" class="formField">
     <h3 class="mt-0">Select company reports</h3>
-    <UploadDocumentsForm ref="uploadDocumentsForm" @documentsChanged="updateSelectedReports" :name="fileUpload" />
+    <UploadDocumentsForm ref="uploadDocumentsForm" @documentsChanged="updateSelectedReports" :name="name" />
   </div>
+  <div v-if="isEuTaxonomy" className="col-9 formFields"></div>
+  <div v-else class="formField">
+    <h3 class="mt-0">Select company reports</h3>
+    <UploadDocumentsForm ref="uploadDocumentsForm" @documentsChanged="updateSelectedReports" :name="name" />
+  </div>
+
```

Comment:
This section confuses me.
What is the difference between the "if" and the "else" case?

Reply:
Not completly sure, but i think the component is used in eu taxonomy upload and also in sfdr, but the styling differs in both cases. I think the if statements are there to define the style which should be used in eu taxonomy upload.

Reply:
I was wondering the same, however it looks like when it is EU Taxonomy, additional stylings are added with an extra DIV ... "col-9 formFields"

So, HTML structure different.

Reply:
This changed a lot the last days. The way it is now, it's ok for me.

The way I see it: Root cause for all the v-ifs and the one v-else is, that the component should look slightly different if used for the eutaxonomy-frameworks.   That's ok for me, if it is necessary due to the FIGMA designs.

Reply:
Yes, exactly its due to differing designs between eu taxonomy and sfdr.